README.md

```diff
@@ -13,3 +13,19 @@ npx install-peerdeps --dev @edx/typescript-config
 ```
 
 This will install the package and all of its peer dependencies.
+
+## Usage
+
+Create file in repository `tsconfig.json`, adding references to the root directory, output directory, and directories to include/exclude in TypeScript compilation.
+
+```json
+{
+	"extends": "@edx/tsconfig",
+	"compilerOptions": {
+        "rootDir": ".",
+        "outDir": "dist"
+	},
+    "include": ["src/**/*"],
+    "exclude": ["dist"]
```

Comment:
[curious] Is it also generally recommended to exclude `node_modules`? Should these `include`/`exclude` be part of the base `tsconfig.json` given that the majority of repos consuming this shared TSConfig will likely want these as defaults?

Reply:
Good call-out!